zokrates_fs_resolver/src/lib.rs

```diff
@@ -41,7 +54,7 @@ mod tests {
     #[test]
     fn valid_path_with_location() {
         let (_, next_location, alias) =
-            resolve(&Some(String::from("./src")), &String::from("lib.rs")).unwrap();
+            resolve(&Some(String::from("./src")), &String::from("./lib.rs")).unwrap();
```

Comment:
Shouldn't we change the source of the other tests as well. No it is kind of trivial that they fail, right?